src/playlist.js

```diff
@@ -0,0 +1,66 @@
+var playItem = require('./playitem.js');
+var setupAutoadvance = require('./autoadvance.js');
+
+var playlistMaker = function(player, plist) {
+  var currentIndex = 0;
+  var autoadvanceTimeout = null;
+
+  var playlist = function playlist(list) {
+    if (typeof list === 'number') {
+      return player.playlist.item(list);
```

Comment:
@heff yeah, I want playlists to feel "core", too. I also agree that `player.playlist().item()` is the more consistent API (though it is slightly trickier to implement). Do you still prefer `player.playlistItem()`?